legacy/app/auth/login.html

```diff
@@ -6,19 +6,19 @@
 
         <div class="form-field">
             <label for="email" translate>user.email</label>
-            <input type="email" id="email" ng-model="email" ng-disabled="processing" placeholder="name@domain.com" required>
+            <input type="email" id="email" ng-model="email" ng-disabled="processing" placeholder="name@domain.com" required data-cy="email">
```

Comment:
Thinking about this, could we switch them up a bit? So that instead of `data-cy=email` we have `data-cy=email-field`
That way they will be a bit more distinct, and we can tell fields from button from text views.
Thoughts?

Reply:
Yes, that makes total sense!